src/Lykke.Service.PaySettlement.AzureRepositories/PaymentRequests/PaymentRequestEntity.cs

```diff
@@ -0,0 +1,235 @@
+                _marketAmount = value;
+                MarkValueTypePropertyAsDirty(nameof(MarketAmount));
+            }
+        }
+
+        private decimal _marketPrice;
+        public decimal MarketPrice
+        {
+            get => _marketPrice;
+            set
+            {
+                _marketPrice = value;
+                MarkValueTypePropertyAsDirty(nameof(MarketPrice));
+            }
+        }
+
+        public string MarketOrderId { get; set; }
+
+        private decimal _transferredAmount;
+        public decimal TransferredAmount
+        {
+            get => _transferredAmount;
+            set
+            {
+                _transferredAmount = value;
+                MarkValueTypePropertyAsDirty(nameof(TransferredAmount));
+            }
+        }
+
+        public string MerchantClientId { get; set; }
+
+        private bool _error;
+        public bool Error
+        {
+            get => _error;
+            set
+            {
+                _error = value;
+                MarkValueTypePropertyAsDirty(nameof(Error));
+            }
+        }
+        public string ErrorDescription { get; set; }
+
+        public PaymentRequestEntity()
+        {
+        }
+
+        public PaymentRequestEntity(IPaymentRequest paymentRequest)
+        {
+            PartitionKey = GetPartitionKey();
+            RowKey = GetRowKey(paymentRequest.Id);
+            Id = paymentRequest.Id;
+            OrderId = paymentRequest.OrderId;
+            MerchantId = paymentRequest.MerchantId;
+            Amount = paymentRequest.Amount;
+            SettlementAssetId = paymentRequest.SettlementAssetId;
+            PaymentAssetId = paymentRequest.PaymentAssetId;
+            DueDate = paymentRequest.DueDate;
+            MarkupPercent = paymentRequest.MarkupPercent;
+            MarkupPips = paymentRequest.MarkupPips;
+            MarkupFixedFee = paymentRequest.MarkupFixedFee;
+            WalletAddress = paymentRequest.WalletAddress;
+            Status = paymentRequest.Status;
+            PaidAmount = paymentRequest.PaidAmount;
+            PaidDate = paymentRequest.PaidDate;
+            PaymentRequestTimestamp = paymentRequest.PaymentRequestTimestamp;
+            TransferToMarketTransactionHash = paymentRequest.TransferToMarketTransactionHash;
+            TransferToMarketTransactionFee = paymentRequest.TransferToMarketTransactionFee;
+            SettlementStatus = paymentRequest.SettlementStatus;
+            MarketAmount = paymentRequest.MarketAmount;
+            MarketPrice = paymentRequest.MarketPrice;
+            MarketOrderId = paymentRequest.MarketOrderId;
+            TransferredAmount = paymentRequest.TransferredAmount;
+            MerchantClientId = paymentRequest.MerchantClientId;
+            Error = paymentRequest.Error;
+            ErrorDescription = paymentRequest.ErrorDescription;
+        }
+
+        internal static string GetPartitionKey()
+        {
+            return "PaymentRequest";
```

Comment:
please select other keys 
1) paymentRequest.MerchantClientId as PK and GUID as RK
2) GUID as PK and this constant as RK